website/install.rst

```diff
@@ -277,6 +277,22 @@ You can customize these options three ways:
 2. Use the UI ``cmake-gui`` or ``ccmake``.
 3. Specify them as command-line arguments when you invoke cmake.
 
+Uninstall
+~~~~~~~~~
+
+If you have installed from source, you can uninstall via: 
+
+.. code-block::
+
+    % cmake --build $builddir --target uninstall
+
+or if using ``make``:
+
+.. code-block::
+
+    % make uninstall
+
+
```

Comment:
This implies that if you still have the exact working source and build tree that built your installation, then running this would remove the installed components.

But if you don't have source, or a build tree, or a build tree that exactly matches the commit and build-time options of your installed OpenEXR, this might be unreliable or entirely unavailable. It's not at all like having an "openexr-uninstall" script installed with the rest of the binary components.

I'm not sure if this satisfies the general requirement to have an uninstall command. Is that sufficient from an OpenSSF standpoint? Do they mean "the person who built it can reverse an install if they catch it before they blow away their build tree?" Or do they mean "any user with permissions to alter the install area can reliably remove all installed components at any time?"

Reply:
@jmertic mentioned on [slack](https://academysoftwarefdn.slack.com/archives/CKB8RR3FT/p1705073047916139) a while back that:

`For source builds, just a note saying "files are created at x, y, z that would need to be removed for uninstall" is sufficent`

So the uninstall target might not be absolutely necessary to satisfy the badge requirements. But I do see this construct adopted by other respectable projects like [pybind11](https://github.com/pybind/pybind11/blob/master/tools/cmake_uninstall.cmake.in) and [c-blosc](https://github.com/Blosc/c-blosc/blob/main/cmake_uninstall.cmake.in) and libjpSo the uninstall target might not be absolutely necessary to satisfy the badge requirements. But I do see this construct adopted by other projects like [pybind11](https://github.com/pybind/pybind11/blob/master/tools/cmake_uninstall.cmake.in) and [c-blosc](https://github.com/Blosc/c-blosc/blob/main/cmake_uninstall.cmake.in) and [libjpeg-turbo](https://github.com/libjpeg-turbo/libjpeg-turbo/blob/main/cmakescripts/cmake_uninstall.cmake.in)
eg-turbo, so it seems reasonable to follow that convention.

I can also add a mention to the obvious - if you installed via a package manager, uninstall with the package manager. This applies only if you've built from source (and still have the configuration files).

Reply:
I think it has value in the sense that if a contributor accidentally installs into its root filesystem, at least there is an uninstall target to undo the damage. When I started compiling stuff a couple of years ago, I did this mistake quite often and `make uninstall` or equivalents saved me a lot of trouble on multiple occasions.

Reply:
I added a bit more clarifying wording.